CHANGELOG.md

```diff
@@ -362,6 +362,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Removed `Strategy.optimizer_zero_grad` ([#11246](https://github.com/PyTorchLightning/pytorch-lightning/pull/11246))
 
+
+- Removed `TPUSpawnStrategy.{tpu_local_core_rank,tpu_global_core_rank}` c `TPUSpawnStrategy.{local_rank,global_rank}` ([#11163](https://github.com/PyTorchLightning/pytorch-lightning/pull/11163))
+
+
+- Removed `SingleTPUStrategy.{tpu_local_core_rank,tpu_global_core_rank}` attributes in favor of `SingleTPUStrategy.{local_rank,global_rank}`([#11163](https://github.com/PyTorchLightning/pytorch-lightning/pull/11163))
```

Comment:
`SingleTPUStrategy` wouldn't need `global_rank` ideally. `local_rank` should suffice.

Reply:
that's true yes, it's just a special case. Should I remove mentioning it in the changelog?

Reply:
Would be better to do a follow up PR to remove the property.

Reply:
Remove why? I think what you mean is to define it as constant 0.